### PR TITLE
Ch-configure-eslint-181339558

### PR DESCRIPTION
**Pull request by gihozoinnocente**
# What does this PR do?
ESLint is the required linter for the project. It'll help check for style violations and flag them.

# Description of Task to be completed?
- [x]  ESLint configuration

# How should this be manually tested?
Run script yarn lint

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Any background context you want to provide?
N/A
# What are the relevant pivotal tracker stories?
[181339558]https://www.pivotaltracker.com/story/show/181339558

# Screenshots (if appropriate)

# Questions:

### CODE REVIEW

**Review by ihonore — APPROVED** [review]

**Review by Dev-nkundineza — APPROVED** [review]

**Review by ericmaniraguha — APPROVED** [review]

**Review by ericmaniraguha — APPROVED** [review]

**Review by ericmaniraguha — APPROVED** [review]

**Review by ericmaniraguha — APPROVED** [review]

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+root = true
```
Comment (CYIMANA-Faisal) [inline .editorconfig:1]: I don't think this file should be pushed.

File: .env [hunk 2]
```diff
@@ -1,4 +1,4 @@
-DB_URL_DEV=postgres://postgres:Password1!@127.0.0.1:5432/dev_barefoot_db
-DB_URL_TEST=postgres://postgres:Password1!@127.0.0.1:5432/test_barefoot_db
-DB_URL_PROD=postgres://postgres:Password1!@127.0.0.1:5432/prod_barefoot_db
+DB_URL_DEV=postgres://postgres:password1!@127.0.0.1:5432/gihozo
```
Comment (CYIMANA-Faisal) [inline .env]: Please don't push environment variables

File: .gitignore [hunk 3]
```diff
@@ -1 +1,12 @@
 node_modules
```
Comment (CYIMANA-Faisal) [inline .gitignore:1]: Work with honore to check the best .gitignore for this project

File: .hound.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+eslint:
```
Comment (CYIMANA-Faisal) [inline .hound.yml:1]: This file should be communicating to the hound CI I cant see the houndCI doing anything

**Review by CYIMANA-Faisal — CHANGES_REQUESTED** [review]
Please work on the reviews

**Review by ericmaniraguha — APPROVED** [review]

**Review by ericmaniraguha — APPROVED** [review]

**Review by ericmaniraguha — APPROVED** [review]

**Review by ericmaniraguha — APPROVED** [review]

**Pull request merged**